openstack/tempest/octavia-tempest/templates/etc/_tempest_extra_options.tpl

```diff
@@ -0,0 +1,144 @@
+[DEFAULT]
+debug = True
+use_stderr = True
+rally_debug = True
+
+[auth]
+use_dynamic_credentials = False
+create_isolated_networks = False
+test_accounts_file = /{{ .Chart.Name }}-etc/tempest_accounts.yaml
+default_credentials_domain_name = tempest
+admin_project_name = {{ default "neutron-tempest-admin1" (index .Values (print .Chart.Name | replace "-" "_")).tempest.admin_project_name }}
+admin_username = {{ default "neutron-tempest-admin1" (index .Values (print .Chart.Name | replace "-" "_")).tempest.admin_name }}
+admin_password = {{ required "A valid .Values.tempestAdminPassword required!" .Values.tempestAdminPassword }}
+admin_domain_name = tempest
+admin_domain_scope = True
+
+[identity]
+uri_v3 = http://{{ if .Values.global.clusterDomain }}keystone.{{.Release.Namespace}}.svc.{{.Values.global.clusterDomain}}{{ else }}keystone.{{.Release.Namespace}}.svc.kubernetes.{{.Values.global.region}}.{{.Values.global.tld}}{{end}}:5000/v3
+endpoint_type = internal
+v3_endpoint_type = internal
+region = {{ .Values.global.region }}
+default_domain_id = {{ .Values.tempest_common.domainId }}
+admin_domain_scope = True
+disable_ssl_certificate_validation = True
+auth_version = v3
+username = {{ default "neutron-tempest-admin1" (index .Values (print .Chart.Name | replace "-" "_")).tempest.admin_name }}
+password = {{ required "A valid .Values.tempestAdminPassword required!" .Values.tempestAdminPassword }}
+domain_name = tempest
+admin_role = admin
+admin_domain_name = tempest
+admin_username = {{ default "neutron-tempest-admin1" (index .Values (print .Chart.Name | replace "-" "_")).tempest.admin_name }}
+admin_password = {{ required "A valid .Values.tempestAdminPassword required!" .Values.tempestAdminPassword }}
+catalog_type = identity
+user_unique_last_password_count = 5
+user_lockout_duration = 300
+user_lockout_failure_attempts = 5
+
+[identity-feature-enabled]
+domain_specific_drivers = True
+project_tags = True
+application_credentials = True
+api_v2 = False
+api_v2_admin = False
+api_v3 = True
+trust = True
+security_compliance = True
+
+[network]
+project_network_cidr = 10.199.0.0/16
```

Comment:
not sure how it was before, but imo values should go into chart values.yaml and be referenced similar to {{ .Values.tempest_common.public_network_id }}